Jonas To-Do-List

Jonash189 opened the pull request:
https://jonas-to-do-list.netlify.app/

Jonash189 committed:
adding styling to the label and making sure accessibilty is at 100%.

Jonash189 committed:
Fixing naming of ToDoList

Jonash189 committed:
Fixing naming bugs

HIPPIEKICK commented on src/Page/MyToDoListPage.jsx:
```diff
@@ -0,0 +1,64 @@
+  border-radius: 100px;
+  overflow: hidden; 
+
+  .content {
```
Why is this not its own styled component? 👀

HIPPIEKICK commented on src/components/AddToDoForm.jsx:
```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
```
No need to import React

HIPPIEKICK commented on src/components/AddToDoForm.jsx:
```diff
@@ -0,0 +1,71 @@
+`;
+
+
+function AddToDoForm() {
```
Don't mix arrow functions with the `function` keyword approach

HIPPIEKICK commented on src/stores/Store.js:
```diff
@@ -0,0 +1,36 @@
+// Store.jsx
```
This is not a component, so I think you can name it lowercase

HIPPIEKICK commented on src/components/counter.jsx:
Is this a component or could it be rewritten as a utility function? Either way, name it approprietly (PascalCase if it's a component)

HIPPIEKICK commented on src/Page/MyToDoListPage.jsx:
The folder name should be in lowercase (and plural like your other folders)

HIPPIEKICK reviewed (changes requested):
Nice to see you're continuing with Styled components 🥳 Your code overall looks good and your Zustand store is well-structured.

**Changes Requested**
Please pay attention to following the clean code guidelines. All small things add up and it will make your project look so much better if you cleaned it up. Let me know if you have any questions

Jonash189 committed:
Implemented all requested changes to improve code clarity and quality.

Jonash189 commented:
@HIPPIEKICK Thank you for the feedback! :)  I’ve addressed all the points mentioned and ensured the code follows the clean code guidelines. Please let me know if there’s anything else I might have missed or if further adjustments are needed.

Jonash189 committed:
Pushing to git

HIPPIEKICK commented on src/App.jsx:
```diff
@@ -1,3 +1,11 @@
-};
+import MyToDoListPage from "./Page/MyToDoListPage";
+
+function App() {
```
Why not an arrow function here?
